neqo-http3-server/src/old_https.rs

```diff
@@ -0,0 +1,180 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![cfg_attr(feature = "deny-warnings", deny(warnings))]
+#![warn(clippy::use_self)]
+
+use std::cell::RefCell;
+use std::collections::HashMap;
+use std::fmt::Display;
+use std::path::PathBuf;
+use std::rc::Rc;
+use std::time::Instant;
+
+use regex::Regex;
+
+use neqo_common::Datagram;
+use neqo_crypto::{AntiReplay, ZeroRttCheckResult, ZeroRttChecker};
+use neqo_http3::Error;
+use neqo_transport::server::{ActiveConnectionRef, Server};
+use neqo_transport::{ConnectionEvent, ConnectionIdManager, Output};
+
+use super::{qns_read_response, Args, HttpServer};
+
+#[derive(Clone, Debug)]
+struct AcceptZeroRttChecker {}
+
+impl ZeroRttChecker for AcceptZeroRttChecker {
+    fn check(&self, _token: &[u8]) -> ZeroRttCheckResult {
+        ZeroRttCheckResult::Accept
+    }
+}
```

Comment:
Why not use the one in neqo-crypto?  It logs with `qwarn!()`, but it is otherwise the same.